src/Dashboard/Http/Controllers/AuthenticateDashboard.php

```diff
@@ -3,27 +3,31 @@
 namespace BeyondCode\LaravelWebSockets\Dashboard\Http\Controllers;
 
 use BeyondCode\LaravelWebSockets\Apps\App;
+use BeyondCode\LaravelWebSockets\Contracts\PushesToPusher;
 use Illuminate\Broadcasting\Broadcasters\PusherBroadcaster;
```

Comment:
This class is not used

![Screenshot from 2020-08-24 18-07-15](https://user-images.githubusercontent.com/832259/91032619-c7067b00-e634-11ea-9dce-f5a7db1ebda5.png)

Reply:
`PushesToPusher` is a trait that gets you the right broadcasting manager based on the current replication driver, so you can call `->broadcast()` onto without having to re-implement it everywhere.

Reply:
The `websockets` driver you are saying about is `RedisPusherBroadcaster`. I already look into the `broadcast()` method of it and I can clearly see that it does what's intended - calling `publish()`.

However, I set up a DashboardLogger instance there and it seems to not log it at all:

```php
$connection = $this->redis->connection(
    config('websockets.replication.redis.connection') ?: 'default'
);

$payload = [
    'appId' => $this->appId,
    'event' => $event,
    'data' => $payload,
    'socket' => Arr::pull($payload, 'socket'),
];

foreach ($this->formatChannels($channels) as $channel) {
    $connection->publish("{$this->appId}:{$channel}", json_encode($payload));

    DashboardLogger::log($this->appId, DashboardLogger::TYPE_REPLICATOR_MESSAGE_PUBLISHED, [
        'channel' => $channel,
        'serverId' => null,
        'payload' => $payload,
        'pubsub' => "{$this->appId}:{$channel}",
    ]);
}
```

Reply:
Can clearly see that my channel does not exist and that's why DashboardLogger doesn't get triggered. 🤔

Reply:
Just found out that Laravel prefixes the Pub/Sub Messages. Why am i such an idiot? 🤦 

It should have been: `1:public`. Need to add this config to the Pub/Sub client.

```
3) "laravel_database_1:public"
4) "{\"appId\":\"1\",\"event\":\"test\",\"data\":{\"asd\":123},\"socket\":null}"
```